src/components/Editor.jsx

```diff
@@ -47,6 +48,10 @@ class Editor extends React.Component {
     this.deEmphasize = this.deEmphasize.bind(this);
   }
 
+  shouldComponentUpdate(nextProps) {
+    return _.isEqual(nextProps, this.props)
```

Comment:
https://facebook.github.io/react/docs/react-component.html#shouldcomponentupdate

What you're saying then is that the Editor component should not call `render()` unless the current and the next props are equal to each other. That doesn't seem correct to me. The child CodeMirror instance will still re-render because its state has changed (i.e. its value), but only re-rendering when the props have not changed seems counter-intuitive, and contrary to what React does.